server/core/server_config_test.go

```diff
@@ -184,6 +187,7 @@ func TestStartWithConfigFileFlag(t *testing.T) {
 
 	server := NewAccountServer()
 	server.InitializeFromFlags(flags)
+	server.config.HTTP.Port = 0 // reset port so we don't conflict
```

Comment:
shouldn't you just pass a flag with zero or -1 to choose port randomly?

Reply:
sadly the flag is host:port and i didn't think to try localhost:0, not sure that even works.